packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -336,6 +336,7 @@ class GlobalObjectKey<T extends State<StatefulWidget>> extends GlobalKey<T> {
 }
 
 /// This class is a work-around for the "is" operator not accepting a variable value as its right operand
+@Deprecated('This class is no longer used in framework and will be deleted soon.')
```

Comment:
Please also add this information to the doc comment.

Reply:
Also please use the syntax described in https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

Reply:
Done